plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -134,6 +134,7 @@ public void testLimitPushdown()
         }
 
         assertThat(query("SELECT name FROM nation LIMIT 30")).isFullyPushedDown(); // Use high limit for result determinism
+        assertThat(query("SELECT name FROM nation LIMIT 3")).isFullyPushedDownRegardlessOfResults();
```

Comment:
This is useful but has a very high chance of making us miss correctness if one slips through during review.

Reply:
That's why the name is discouragingly long